src/components/form/form-directive.ts

```diff
@@ -2,7 +2,7 @@ import 'angular';
 let JsonRefs = require('json-refs');
 
 import {IPathResolver} from '../services/pathresolver/jsonforms-pathresolver';
-import {IUISchemaGenerator} from '../generators/generators';
+import {UiSchemaRegistry} from '../ng-services/uischemaregistry/uischemaregistry-service';
```

Comment:
We should perhaps agree on how we want to treat abbreviations (that is, `Ui` or `UI`), but that's another discussion topic.